### DISCUSSION TITLE
Maxpower option: how to disable the 5 attempts?

### DISCUSSION THREAD

**Discussion by drakkons** [arendst/Tasmota#18792] [Q&A]
Hi, 
I can't figure out if it's a bug or if I'm not trying to figure out how to disable the feature. Is there a way, after having set the maxpower function and the maxpowerhold function, NOT to make 5 attempts to definitively set the device switch off? After verifying that the watts have been exceeded, I would like the device to turn off the switch definitively, without making other "attempts".
I tried everything but couldn't come up with a solution
 thanks a lot to everyone!

**Comment by tobi320** [comment]
could you solve the problem?please tell me how

**Reply by sfromis** [reply]
Tried setting [MaxPowerWindow](https://tasmota.github.io/docs/Commands/#maxpowerwindow) to 0?

**Comment by tobi320** [comment]
yes but the device turns on again immediately after it is switched off

**Reply by sfromis** [reply]
Well, the general answer to how to automate in a way not covered by standard features is to create rules (or similar).

**Reply by tobi320** [reply]
thanks,can you help me creating that rule?
